lib/utils.sh

```diff
@@ -0,0 +1,22 @@
+function print_error() {
+   echo "$1"
+   echo ""
+   echo "$CMD_USAGE"
+   exit 1
+}
+
+function override_aws_credentials() {
+    AWS_VARS="AWS_PROFILE AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS AWS_SECRET_ACCESS AWS_REGION"
+
+    for VAR_NAME in ${AWS_VARS}; do
+        FIRST_VAR_NAME="$1_$VAR_NAME"
+        SECOND_VAR_NAME="$2_$VAR_NAME";
+
+        if [[ -n ${!FIRST_VAR_NAME} ]]; then
```

Comment:
Could probably check if `$1` and `$2` is set that the top of the function and error there instead.